server/service/IndividualServicesService.js

```diff
@@ -151,20 +155,41 @@ exports.putLiveAirInterfaceTransimitterIsOn = function(body,mountName,uuid,local
  * customerJourney String Holds information supporting customer’s journey to which the execution applies
  * returns inline_response_200_1
  **/
-exports.putLiveControlConstructExternalLabel = function(body,mountName,user,originator,xCorrelator,traceIndicator,customerJourney) {
-  return new Promise(function(resolve, reject) {
-    var examples = {};
-    examples['application/json'] = {
-  "response-code" : 204
-};
-    if (Object.keys(examples).length > 0) {
-      resolve(examples[Object.keys(examples)[0]]);
-    } else {
-      resolve();
+exports.putLiveControlConstructExternalLabel = async function (body, mountName, user, originator, xCorrelator, traceIndicator, customerJourney) {
+  let responseCode = "";
+  try {
+    let mountNamevalue = mountName;
+    let requestBody = body;
+
+    let eatlRequestHeaders = {
+      user: user,
+      originator: originator,
+      xCorrelator: xCorrelator,
+      traceIndicator: traceIndicator,
+      customerJourney: customerJourney
+    };
+
+    let base64Auth = await OpenDayLightAdapter.getAuthorizationAsync()
+    if (base64Auth) {
+      let httpRequestHeader = {
+        "Authorization": base64Auth
+      }
+      responseCode = await PutToLiveControlConstructExternalLabelCausesWritingIntoDevice(mountNamevalue, requestBody, httpRequestHeader, eatlRequestHeaders)
```

Comment:
instead of defining it in a separate function , the content defined in the function PutToLiveControlConstructExternalLabelCausesWritingIntoDevice shall be included here itself as its not going to be used by any other services.